DABI.js

```diff
@@ -0,0 +1,677 @@
+{
+    "translatorID": "5cf8bb21-e350-444f-b9b4-f46d9fab7827",
+    "label": "DABI",
+    "creator": "Jens Mittelbach",
+    "target": "dabi.ib.hu-berlin\\.de/.*?",
```

Comment:
Use `"^https?://dabi\\.ib\\.hu-berlin\\.de/"` (there's no point in adding optional matches to the end, since they don't restrict anything)